pkg/infra/iac2/resource_creation_template.go

```diff
@@ -178,6 +183,54 @@ func parseTS(val reflect.Value) (string, error) {
 	return "", fmt.Errorf("invalid template value detected: %s: %v: template values must implement the templateValue interface", val.Kind().String(), val.Interface())
 }
 
+func appliedOutputsToString(outputs []AppliedOutput) string {
```

Comment:
I thought Jordan removed this - is this a merge/rebase error?

Reply:
I added it back. There are still cases where something like this is needed. An example of this is where you need to use a pulumi output as an attribute name in an object.

here's an example of how it's used:
```ts
const myTaskExecutionRole = new aws.iam.Role(`my-task-ExecutionRole`, {
    assumeRolePolicy: pulumi.jsonStringify(
      // we invoke .apply() on a specific applied output instead of using pulumi.All() when there's only 1 applied output
      myEksCluster.url.apply(cluster_oidc_url => {
          return {
              Version: `2012-10-17`,
              Statement: [{
                  Effect: `Allow`,
                  Action: [`sts:AssumeRoleWithWebIdentity`],
                  Principal: {
                      Federated: myEksCluster.arn,
                  },
                  Condition: {
                      StringEquals: {
                          // cluster_oidc_url needs to be an actual string here
                          [`${cluster_oidc_url}:sub`]: `system:serviceaccount:default:my-task`,
                          [`${cluster_oidc_url}:aud`]: `sts.amazonaws.com` 
                      },
                  },
              }],
          }
    })),
```